doc/conf.py

```diff
@@ -372,6 +372,7 @@
     ('boards/arm/nucleo_f207zg/doc/nucleo_f207zg', 'boards/arm/nucleo_f207zg/doc/index'),
     ('boards/arm/nucleo_f334r8/doc/nucleof334r8', 'boards/arm/nucleo_f334r8/doc/index'),
     ('boards/arm/nucleo_f401re/doc/nucleof401re', 'boards/arm/nucleo_f401re/doc/index'),
+    ('boards/arm/nucleo_f410rb/doc/nucleof410rb', 'boards/arm/nucleo_f410rb/doc/index'),
```

Comment:
This change is not required for newly introduced boards

Reply:
Should I remove this?